### DISCUSSION TITLE
What is the maximum file length / file size you have been able to transcribe using nf-whisper.

### DISCUSSION THREAD

**Discussion by mpampuch** [mribeirodantas/nf-whisper#11] [Q&A]
Relatedly, would you recommend converting mp4 files to mp3 files first before running nf-whisper

**Comment by mribeirodantas** [comment]
I'm not sure about file length / file size but recently I used nf-whisper on [all these talks](https://www.youtube.com/watch?v=mvhMnNl9lsQ&list=PLPZ8WHdZGxmXuWK4X5yJyFDqNzc3GhVBz). The transcriptions weren't perfect but good enough, given some manual curation.

There's a step in the pipeline to extract the audio from the YouTube video URL, but yes, it's quicker if you provide it directly with the mp3 file, yes
